wdl-engine/src/inputs.rs

```diff
@@ -614,15 +624,16 @@ impl Inputs {
     /// Parses an inputs file from the given file path.
     ///
     /// The format (JSON or YAML) is determined by the file extension:
+    ///
     /// - `.json` for JSON format
     /// - `.yml` or `.yaml` for YAML format
     ///
     /// The parse uses the provided document to validate the input keys within
     /// the file.
```

Comment:
The `wdl-cli` implementation doesn't care about the extension. Should this be updated to use the same logic? (Attempt to parse as both, ignoring the extension)

Reply:
I agree with Peter, so I'll keep it as is.